lib/ecto_enum.ex

```diff
@@ -143,4 +143,13 @@ defmodule EctoEnum do
     def dump(_), do: :error
   end
 
+  def validate_enum(changeset, field) do
+    type = changeset.types[field]
+    value = get_field(changeset,field)
+
+    case type.valid_value?(value) do
+      true -> changeset
+      _ -> add_error(changeset, field, "Value #{value} is not member of #{field} enum")
```

Comment:
Can you make the error message here configurable by adding a 3rd param to the function? Make the 3rd param a function that accepts a`value` and a `field` as a paramter and returns a string error message.